docs/Reference/API-Methods.md

```diff
@@ -3967,10 +3967,11 @@ None
 
 ### eea_sendRawTransaction
 
-Creates a [private transaction](../Concepts/Privacy/Privacy-Overview.md) from a signed transaction, generates the transaction hash 
-and submits it to the transaction pool, and returns the transaction hash of the Privacy Marker Transaction.
+Distributes the [private transaction](../HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md), 
+generates the transaction hash and submits it to the transaction pool, and returns the transaction hash of the Privacy Marker Transaction.
```

Comment:
I think this is a little confusing. Is this the transaction hash of the private transaction? No - it is the transaction hash of the privacy marker transaction that is sent to the transaction pool. And then link to the part of the doc where you define PMT.

Reply:
Fixed